main.py

```diff
@@ -49,41 +56,14 @@
     # Set the drive base speed and turn rate.
     robot.drive(DRIVE_SPEED, turn_rate)
     # If the ultrasonic sensor sees an obstacle, stop the robot and beep then turn around.
-    if obstacle_sensor.distance() < 200:
+    if obstacle_sensor.distance() < 0:
```

Comment:
distance <0 does not make sense, the robot should have little playroom to turn around